aad/client.bal

```diff
@@ -0,0 +1,423 @@
+    # 
+    # + info - The basic information about a user
+    # + return - A record of type `User` if sucess. Else `error`.
+    @display {label: "Create User"}
+    remote isolated function createUser(@display {label: "New User Information"} NewUser info) returns 
+                                        User|error {
+        string path = check createUrl([USERS]);
+        json payload = check info.cloneWithType(json);
+        return check self.httpClient->post(path, payload, targetType = User);
+    }
+
+    # Get information about a user.
+    # 
+    # + userId - The object ID if the user
+    # + queryParams - Optional query parameters. This method support OData query parameters to customize the response.
+    #                 It should be an array of type `string` in the format `<QUERY_PARAMETER_NAME>=<PARAMETER_VALUE>`
+    #                 **Note:** For more information about query parameters, refer here: 
+    #                   https://docs.microsoft.com/en-us/graph/query-parameters
+    # + return - A record of type `User` if sucess. Else `error`.
+    @display {label: "Get User"}
+    remote isolated function getUser(@display {label: "User ID"} string userId, 
+                                     @display {label: "Optional Query Parameters"} string[] queryParams = []) 
+                                     returns User|error {
+        string path = check createUrl([USERS, userId], queryParams);   
+        return check self.httpClient->get(path, targetType = User);
+    }
+
+    # Update information about a user.
+    # 
+    # + userId - The object ID if the user
+    # + info - The information to update
+    # + return - `nil` if sucess. Else `error`.
+    @display {label: "Update User"}
+    remote isolated function updateUser(@display {label: "User ID"} string userId, 
+                                        @display {label: "User Update Information"} UpdateUser info) 
+                                        returns error? {
+        string path = check createUrl([USERS, userId]); 
+        json payload = check info.cloneWithType(json);   
+        http:Response response = check self.httpClient->patch(path, payload);
+        _ = check handleResponse(response);
+    }
+
+    # List information about users.
+    # 
+    # + queryParams - Optional query parameters. This method support OData query parameters to customize the response.
+    #                 It should be an array of type `string` in the format `<QUERY_PARAMETER_NAME>=<PARAMETER_VALUE>`
+    #                 **Note:** For more information about query parameters, refer here: 
+    #                   https://docs.microsoft.com/en-us/graph/query-parameters
+    # + return - A stream of type `User` if sucess. Else `error`.
+    @display {label: "List Users"}
+    remote isolated function listUsers(@display {label: "Optional Query Parameters"} string[] queryParams = []) 
+                                       returns stream<User, error?>|error {
```

Comment:
Can you add  display annotation for stream?

Reply:
Added them. Thanks.